src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="navigation">
+        <a href="index.html" class="icon">
+          <img src="images/logo.png"
+          class="nav"
+          alt="Mate Academy">
+        </a>
+        <ul class="nav__list">
+          <li>
+            <a href="#Apple" class="margin__nav is-active">Apple</a>
```

Comment:
Where is the margin block. Fix everywhere
```suggestion
            <a href="#Apple" class="nav__link is-active">Apple</a>
```